docs/source/reference/yaml-spec.rst

```diff
@@ -317,6 +326,15 @@ OR
       idle_minutes: 10
       down: true  # Use autodown instead of autostop
 
+OR
+
+.. code-block:: yaml
+
+  resources:
+    autostop:
+      idle_minutes: 10
+      wait_for: none  # Stop after 10 minutes, regardless of running jobs or SSH connections
```

Comment:
can we add this to the entire YAML spec at the top as well?

Reply:
Added